config/default.yml

```diff
@@ -3963,10 +3963,10 @@ Style/EmptyStringInsideInterpolation:
   Description: 'Checks for empty strings being assigned inside string interpolation.'
   StyleGuide: '#empty-strings-in-interpolation'
   Enabled: pending
-  EnforcedStyle: ternary
+  EnforcedStyle: trailing_conditional
   SupportedStyles:
-    - ternary
     - trailing_conditional
+    - ternary
```

Comment:
Why reverse the order here?

Reply:
Since this is a cop that hasn't been released yet, the entries have been reordered to match the order used in the documentation examples and test code.